package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "simple-icons",
-  "version": "1.9.13",
+  "version": "1.9.18",
```

Comment:
Can you revert this change? Even if it should be updated, I'd prefer that not to be part of this PR 😄